_posts/0000-01-02-andersonavery.md

```diff
@@ -0,0 +1 @@
+here is some test code
```

Comment:
---
layout: slide
title: "Welcome to our second slide!"
---
Your text
Use the left arrow to go back!